README.md

```diff
@@ -0,0 +1,48 @@
+# `subra`
+> project subra v1.0-SNAPSHOT
+
+You'll need at least Java 1.8 ([jre](https://www.java.com/download/)
+/[jdk](http://www.oracle.com/technetwork/java/javase/downloads/index-jsp-138363.html)) 
+to run `subra`.
+
+## How to use
+
+In order to use `subra` clone this repository:
+
+```
+  git clone git@github.com:delta-leonis/subra.git
+```
+
+## Documentation
+
+The javadoc for the current code on `master` can be found on https://delta-leonis.github.io/subra/
+
+## Building
+
+Make sure you have `gradle>=v2.10` installed. Run the following to build the application:
+
+```
+  gradle build
+```
+
+## Copyright
+
+This project is licensed under the AGPL version 3 license (see LICENSE).
+
+```
+subra - project subra v1.0-SNAPSHOT
```

Comment:
sure you want to have the (wrong) version here? I'd suggest removing th eversion at all.